packages/webview_flutter/webview_flutter/ios/Classes/FlutterWebView.m

```diff
@@ -391,25 +386,15 @@ - (void)updateAutoMediaPlaybackPolicy:(NSNumber*)policy
     case 0:  // require_user_action_for_all_media_types
       if (@available(iOS 10.0, *)) {
         configuration.mediaTypesRequiringUserActionForPlayback = WKAudiovisualMediaTypeAll;
-      } else if (@available(iOS 9.0, *)) {
-        configuration.requiresUserActionForMediaPlayback = true;
       } else {
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wdeprecated-declarations"
-        configuration.mediaPlaybackRequiresUserAction = true;
-#pragma clang diagnostic pop
```

Comment:
Optional: We cut down on warnings for users who build for 10+ as a minimum by moving these pragmas to the iOS 9 code instead of removing them entirely.